fabric-lifecycle-events-v1/src/main/java/net/legacyfabric/fabric/api/event/lifecycle/v1/ServerTickEvents.java

```diff
@@ -33,7 +34,7 @@ private ServerTickEvents() {
 	public static final Event<StartTick> START_SERVER_TICK = EventFactory.createArrayBacked(StartTick.class, callbacks -> server -> {
 		if (EventFactory.isProfilingEnabled()) {
 			final Profiler profiler = server.profiler;
-			profiler.push("fabricStartServerTick");
+			profiler.swap("fabricStartServerTick");
```

Comment:
why?

Reply:
because the invoker injects right before a profiler call